projects/tinkerbell/pbnj/Makefile

```diff
@@ -0,0 +1,35 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+ARTIFACTS_BUCKET=s3://projectbuildpipeline-857-pipelineoutputartifactsb-10ajmk30khe3f
+IPMITOOL_CLONE_URL?=https://github.com/ipmitool/ipmitool.git
+ifeq ($(CODEBUILD_CI),true)
```

Comment:
I dont love this... What if we added a function in Common.mk that takes a repo/component or whatever and returns the clone_url?  We could use it for the main clone_url as well and "hide" the codebuild_ci part in one place?

Reply:
Done